src/components/Navbar.jsx

```diff
@@ -1,96 +1,105 @@
-import React, { useState } from 'react';
-import { FaBars, FaTimes } from 'react-icons/fa';
+import React, { useState } from "react";
+import { FaBars, FaTimes } from "react-icons/fa";
 
 const navlinks = [
```

Comment:
home
about us -> href: /#about
suggestions
template
features

change navbar content to above